services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/Enums.kt

```diff
@@ -35,9 +35,10 @@ enum class Linjaautoliikenneluokittelu(override val value: Int) : Luokittelu {
     KAMPPI_RAUTATIENTORI_MANNERHEIMINTIE_KAISANIEMENKATU_HAMEENTIE_TAI_YLI_20_VUOROA_RUUHKATUNNISSA(
         5
     ),
-    RUNKOLINJA_TAI_ENINTAAN_20_VUOROA_RUUHKAAIKANA(4),
-    RUNKOLINJAMAINEN_LINJA_TAI_ENINTAAN_10_VUOROA_RUUHKAAIKANA(3),
-    ENINTAAN_5_VUOROA_RUUHKAAIKANA_TAI_LINJOJA_MUUNA_AIKANA(2),
+    RUNKOLINJA_TAI_ENINTAAN_20_VUOROA_RUUHKATUNNISSA(4),
+    ENINTAAN_10_VUOROA_RUUHKATUNNISSA(3),
+    EI_RUUHKAAIKANA(2),
+    MAHDOLLINEN_POIKKEUSREITTI(1),
     EI_VAIKUTA_LINJAAUTOLIIKENTEESEEN(0)
```

Comment:
The enum names could be made simpler, with more detailed descriptions as documentation.
```kotlin
    /**
     * Rautatientori, Kamppi, Mannerheimintie, Kaisaniemenkatu, Hämeentie or over 20 buses per hour.
     */
    TARKEIMMAT_JOUKKOLIIKENNEKADUT(5),
    /** Trunk line (runkolinja) or over 10 buses per hour. */
    RUNKOLINJA(4),
    /** At most 10 buses per hour during rush hour. */
    VUOROJA_RUUHKAAIKANA(3),
    /** No regular buses during rush hour. The might be buses at other times, like night buses. */
    EI_VUOROJA_RUUHKAAIKANA(2),
```

Reply:
I renamed enum values and added comments.